tox.ini

```diff
@@ -34,15 +35,40 @@ basepython =
 extras=
     test
     docs: doc
-whitelist_externals=make
+allowlist_externals=make
 passenv=PYTEST_ADDOPTS
 
-[testenv:lint]
-basepython=python
-extras=lint
+[common-lint]
+extras=dev
 commands=
     mypy --follow-imports=silent -p eth_abi.utils --config-file {toxinidir}/mypy.ini
     flake8 --extend-ignore=W504,E203,W503 {toxinidir}/eth_abi {toxinidir}/tests
     isort --recursive --check-only --diff {toxinidir}/eth_abi {toxinidir}/tests
     black eth_abi/ tests/ setup.py --check
     pydocstyle {toxinidir}/eth_abi {toxinidir}/tests --add-ignore=D415
+
+
+[testenv:py{37,38,39,310,311}-lint]
+extras: {[common-lint]extras}
+commands: {[common-lint]commands}
+
```

Comment:
We should still keep a `lint` setup for the `Makefile` so we can run `make lint` locally with `python` as `basepython`. I think this should do:

```ini
[testenv:lint]
basepython: python
extras: {[common-lint]extras}
commands: {[common-lint]commands}
```

**Optional nit:** The lint command `tox -elint` in the `Makefile` should be `tox -e lint`. Might be a reasonable / appropriate time to fix?

Reply:
> We should still keep a lint setup for the Makefile so we can run make lint locally with python as basepython.

Aha! I couldn't remember what that was for. Nice catch. 

> The lint command tox -elint in the Makefile should be tox -e lint.

Will do!